src/UniGetUI.Core.Data/CoreData.cs

```diff
@@ -287,7 +284,8 @@ private static string GetNewDataDirectoryOrMoveOld(string old_path, string new_p
                     return new_path;
                 }
             }
-            else if (/*Directory.Exists(new_path)*/Directory.Exists(old_path))
```

Comment:
Removing the _redundant_ else, the system will always check for the 4 conditions. If the else is present, once one condition is met the other ones will not be checked.

Since those are filesystem calls, and they are used quite a few times in UniGetUI, I would prefer to maintain the _redundant_ else statement.